db/migrations/0005.add_job_run_table.py

```diff
@@ -0,0 +1,18 @@
+#
+# file: migrations/0005.add_job_run_table.py
+#
+from yoyo import step
+
+__depends__ = {'0004.add_sis_id'}
+
+step('''
+    CREATE TABLE IF NOT EXISTS job_run
+    (
+        id INTEGER NOT NULL UNIQUE AUTO_INCREMENT,
```

Comment:
There will be multiple jobs running to get data from different sources, and those job status will all be stored in this table. 

How about adding a column here for the job_name, e.g. "course_inventory", "Zoom", "MiVideo", etc.

Reply:
@zqian, I can do this, but if so, I do not want to include the `unizin_data_updated_at` here, since some jobs will not require this and I'd rather not have a bunch of null values. How about I create a `data_source_status` table and relate it many to one to `job_run`? Something like this:
![Screen Shot 2020-03-27 at 10 07 37 AM](https://user-images.githubusercontent.com/35741256/77764340-e4e29c00-7012-11ea-9f10-ba1c605d49d5.png)

Reply:
Yeah I think this should be more generic. In MyLA we're using the django cron and while that package is unmainted and has a few issues I think their data model is pretty good and does what we need here. We probably don't really two separate tables. Aren't all of the values always going to be populated? 

```
    """
    Keeps track of the cron jobs that ran etc. and any error
    messages if they failed.
    """
    code = models.CharField(max_length=64, db_index=True)
    start_time = models.DateTimeField(db_index=True)
    end_time = models.DateTimeField(db_index=True)
    is_success = models.BooleanField(default=False)
    message = models.TextField(default='', blank=True)  # TODO: db_index=True

    # This field is used to mark jobs executed in exact time.
    # Jobs that run every X minutes, have this field empty.
    ran_at_time = models.TimeField(null=True, blank=True, db_index=True, editable=False)
```

https://github.com/Tivix/django-cron/blob/master/django_cron/models.py

Reply:
I think if we the start/end times in this database it would make it unnecessary to do the logging in the future too. The nice thing about the django cron is to create a job you just extend [CronJobBase](https://github.com/Tivix/django-cron/blob/7fa5c2d2835520d8c920f3c103937ab292d9e545/django_cron/__init__.py#L43) and implement a do() as your method. This could be implemented as part of #33 for a similar pattern to ensure the job_run table is populated as expected.

Reply:
@jonespm, from my perspective, no, sometimes we won't  have a `data_source_updated_at` value, since we more or less have to assume API data is current. I was trying to handle the possibility of having multiple data sources in a job (which mine currently does).

I don't know how well the CronJob model does this, though some of those fields might be useful.